cmake/modules/AddCephTest.cmake

```diff
@@ -2,7 +2,36 @@
 
 #adds makes target/script into a test, test to check target, sets necessary environment variables
 function(add_ceph_test test_name test_path)
-  add_test(NAME ${test_name} COMMAND ${test_path} ${ARGN}
+  # crimson seastar unittest always run with --memory XXXM --smp N
+  string(FIND "${ARGN}" "--memory" mem_pos)
+  string(FIND "${ARGN}" "--smp" smp_pos)
+  if((NOT mem_pos EQUAL -1) AND (NOT smp_pos EQUAL -1))
```

Comment:
I think it is probably not a good assumption that all crimson tests always run with "--memory", and this feature doesn't seem to be related to the memory configuration. Is it fine to drop the memory related checks?

Otherwise, the cpu allocation policy below seems good to me.

Reply:
Ok, I will remove the `--memory` checks